Kernel/Net/E1000NetworkAdapter.cpp

```diff
@@ -117,6 +117,8 @@ namespace Kernel {
 #define INTERRUPT_TXD_LOW (1 << 15)
 #define INTERRUPT_SRPD (1 << 16)
 
+#define PCI_VENDOR_INTEL 0x8086
```

Comment:
Since this macro is suitable for other purposes (also in the near future, a hint for my PR on Intel graphics), let's start with a file for such macros. This is also doesn't fill quite right to put this macro here, as E1000-compatible NICs (and there are lots of these) are almost guaranteed to be manufactured by Intel.

Reply:
I've created a new file for those (`Kernel/PCI/IDs.h`) and moved all existing vendor and device IDs there.